src/main/java/com/minecolonies/coremod/colony/requestsystem/resolvers/StandardPlayerRequestResolver.java

```diff
@@ -71,8 +71,9 @@ public boolean canResolve(@NotNull final IRequestManager manager, final IRequest
         return null;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Return an empty collection instead of null. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1168)